.github/workflows/test.yml

```diff
@@ -3,7 +3,9 @@ name: ovn-ci
 on:
   merge_group:
   pull_request:
-    branches: [ master ]
+    branches:
+    - master
+    - release-1.0
```

Comment:
Adding that change in master branch in this PR:  https://github.com/ovn-org/ovn-kubernetes/pull/4468
@tssurya